index.html

```diff
@@ -1,12 +1,64 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Нёрдс</title>
-  </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
+    <header>
+      <nav>
+        <a href="main.html">
+          <img src="img/index-logo.svg" width="368" height="153" alt="Логотип студии «Нёрдс»">
+        </a>
+        <ul>
+          <li><a>Студия</a></li>
+          <li><a>Клиенты</a></li>
+          <li><a>Магазин</a></li>
+          <li><a>Контакты</a></li>
+          <li><a>Корзина</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <div>
+          <a>
+            <img src="img/websites.jpg" width="286" height="164" alt="websites">
+          </a>
```

Comment:
Ссылка не нужна. Просто:
```suggestion
            <img src="img/websites.jpg" width="286" height="164" alt="websites">
```